docs/source/cuda_ann_benchmarks.md

```diff
@@ -60,6 +60,143 @@ There are 4 general steps to running the benchmarks:
 3. Search Using Built Index
 4. Evaluate Result
 
+We provide two methods for running benchmarks, using:
+1. Python
+2. bash
+
+### Python
+#### End-to-end example: Million-scale
+```bash
+# All scripts are assumed to be run in directory raft/cpp/bench/ann
```

Comment:
I think the scripts might be a little more clearner to run if we assumed they were run from the raft root and maybe existed in `$RAFT_HOME/scripts/ann-benchmarks/`. What do you think? I'm just trying to make them as simple as possible to run for any user after they are built. I think we might also start pushing them as a conda package so that users can run them right out of the box.